demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1927,16 +1934,27 @@ export class DemoMeetingApp
       li.className = 'list-group-item d-flex justify-content-between align-items-center';
       li.appendChild(document.createElement('span'));
       li.appendChild(document.createElement('span'));
+      li.appendChild(document.createElement('input'));
       roster.appendChild(li);
     }
     while (roster.getElementsByTagName('li').length > newRosterCount) {
       roster.removeChild(roster.getElementsByTagName('li')[0]);
     }
+
+    // Delete the selected attendees that are no longer part  of the roster
+    for (let attendeeId of this.selectedAttendees) {
+        if (this.roster[attendeeId] == undefined) {
+          this.selectedAttendees.delete(attendeeId);
+          this.meetingLogger.error(attendeeId);
```

Comment:
Why do we log error here?

Reply:
Good catch! left that accidentally there while debugging. Will remove it.